src/routes/RecipeDetails.js

```diff
@@ -28,8 +28,11 @@ class RecipeDetails extends React.Component {
           alt={recipe.title}
           className="pb-4"
         />
-        <div className="text-left border-bottom pb-4">
-          {recipeDetail?.description}
+        <div 
+          className="text-left border-bottom pb-4" 
+          dangerouslySetInnerHTML={
+            {__html: recipeDetail?.description}
+          }>
```

Comment:
This works fine. It's not very pretty though. FYI, there is a library that you can use. [react-render-html](https://www.npmjs.com/package/react-render-html). If you would use that, you can import a function `renderHTML`
```js
import React from 'react'
import PropTypes from 'prop-types'
import renderHTML from 'react-render-html';

class RecipeDetails extends React.Component {
  ...
  render() {
    ...
    <div className="content-container border-bottom pb-4">
      {renderHTML(this.state.recipe.recipeDetail.description)}
    </div>
    ...
  }
}
```
Not mandatory though .. your solution works just fine. Feel free to change or to leave as it is. 👍

Reply:
Thank you! I saw the library, yes.. if you don't mind, I'll leave it for next time